cmd/client/printer.go

```diff
@@ -88,8 +93,24 @@ func PrintShard(shard *metadata.Shard) {
 				slotRangesToString(shard.SlotRanges),
 				shard.ImportSlot,
 				shard.MigratingSlot,
+				status,
 			},
 		})
 	}
 	t.Render()
 }
+
+func isNodeAlive(node metadata.NodeInfo, cluster *metadata.Cluster) bool {
+
+	lastHeartbeat := node.CreatedAt
```

Comment:
The node.Created means the time of creating the node time point instead of the heartbeat time. We should use the ping command to check whether the node is alive or not.